core/modules/user/user.theme.inc

```diff
@@ -386,3 +386,68 @@ function template_preprocess_user_profile(&$variables) {
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid;
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid . '__' . $view_mode;
 }
+
+/**
+ * Preprocess variables for user-simplified-page.tpl.php.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - form: The form structure to be rendered.
+ *   - links: A render array containing the links to other related forms.
+ *
+ * @see user-simplified-page.tpl.php
+ */
+function template_preprocess_user_simplified_page(&$variables) {
+  $form = $variables['form'];
+  if ($form['#form_id'] == 'user_login') {
+    $page_title = t('Login');
+  }
+  elseif ($form['#form_id'] == 'user_pass') {
+    $page_title = t('Reset password');
+  }
+  elseif ($form['#form_id'] == 'user_register_form') {
+    $page_title = t('Create new account');
+  }
+  
+  $variables['user_login_appearance_setting'] = config_get('system.core', 'user_login_appearance_setting');
+  $variables['user_login_page_simplified'] = $variables['user_login_appearance_setting'] == 'simplified';
+  $variables['logo_image'] = '';
+
+  if ($variables['user_login_page_simplified']) {
+    if ($logo = backdrop_get_logo()) {
+      $variables['logo_image'] = theme('image', array(
+        'path' => $logo,
+        'alt' => $page_title,
+      ));
+    }
+    $variables['site_name'] = config_get_translated('system.core', 'site_name');
+    $variables['front_page'] = url();
+    $variables['page_title'] = $page_title;
+    $variables['messages'] = theme('status_messages');
+  }
+  backdrop_add_css(backdrop_get_path('module', 'user') . '/css/user-simplified-page.css');
+}
+
+/**
+ * Returns HTML for user forms with links instead of tabs.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - form: A render array of the form to be rendered
+ *     (e.g. login or password reset).
+ *   - links: An item_list render array with the local tasks as links. 
+ *
+ * @return string
+ *   The themed HTML string with the links.
+ * 
+ * @ingroup themeable
+ */
+function theme_user_links_page($variables) {
+  $output = '<div class="user-form">';
```

Comment:
I notice we have this wrapper around the form here, but not in `user-simplified-page.tpl.php`. We also don't have it on the "normal" tab-versions of these pages. It may be better to remove this wrapper and add it as `$form['#attributes']['class']` instead in all 3 forms so they can all be targetted with that shared selector. Though it should probably be given a better class name, maybe `user-common-login-form`?

Reply:
Since these links are not "login" links (they include password reset and create account), I'll change to "user-form-links". If this is not OK let me know.

Reply:
I've removed this div, since it's not in the simplified form, and wasn't there in the current login forms. I don't think a wrapper class is really needed, so I haven't added it to the form attributes either.